src/Host/Broker/Impl/UserProfile/DeleteProfileController.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Common.Core;
+using Microsoft.R.Host.Broker.Security;
+using Microsoft.R.Host.Broker.Sessions;
+using Microsoft.R.Host.Protocol;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.R.Host.Broker.UserProfile {
+    [Authorize(Policy = Policies.RUser)]
+    [Route("/deleteuser")]
```

Comment:
Per REST, the route itself should just be `/user` or something similar (maybe `/profile`? we might have `/users` later as an admin endpoint for user management) - the action is represented as HTTP method. We might add an HTTP GET endpoint here later for things like user querying info about his profile.